static/src/js/components/GranuleResults/GranuleResultsList.js

```diff
@@ -85,6 +85,9 @@ export class GranuleResultsList extends PureComponent {
       granuleIds = difference(allGranuleIds, excludedGranuleIds)
     }
 
+    // Determine the correct granule count based on granules that have been removed
+    const granuleCount = (hits - excludedGranuleIds.length)
```

Comment:
If you are now excluding the granules in the request, do you need to subtract the excluded count like this? Seems like the hits in the request should have the correct value, no?

Reply:
I'm not excluding them in the requests in the UI, I only add them to the params when the download is submitted and stored in the database. Those granule params are only used when displaying data for the download access methods and when the orders are submitted to CMR and ESI.